doc/source/whatsnew/v1.5.0.rst

```diff
@@ -908,7 +908,7 @@ Indexing
 - Bug in :meth:`NDFrame.xs`, :meth:`DataFrame.iterrows`, :meth:`DataFrame.loc` and :meth:`DataFrame.iloc` not always propagating metadata (:issue:`28283`)
 - Bug in :meth:`DataFrame.sum` min_count changes dtype if input contains NaNs (:issue:`46947`)
 - Bug in :class:`IntervalTree` that lead to an infinite recursion. (:issue:`46658`)
--
+- Bug in :meth:`DataFrame.loc` when creating a new element on a :class:`Series` with dtype :class:`CategoricalDtype` (:issue:`47677`)
```

Comment:
release note not needed if only a fix for issue on main. If changing behavior from 1.4.3, as suggested in https://github.com/pandas-dev/pandas/pull/47751#issuecomment-1186244051 then will need a release note covering just that change.